python/mxnet/libinfo.py

```diff
@@ -96,10 +96,17 @@ def find_include_path():
             logging.warning("MXNET_INCLUDE_PATH '%s' doesn't exist", incl_from_env)
 
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
-    incl_path = os.path.join(curr_path, '../../include/')
-    if not os.path.isdir(incl_path):
-        raise RuntimeError('Cannot find the MXNet include path.\n')
-    return incl_path
+    # include path in pip package
+    incl_path = os.path.join(curr_path, './include/')
```

Comment:
I think you do not need './' before 'include/'. But it is ok to keep it.

Reply:
updated